e2e/cloud_manager/decryption_kmip/test2-input

```diff
@@ -0,0 +1,4 @@
+{"ts":{"$date":{"$numberLong":"1648685783241"}},"version":"0.0","compressionMode":"zstd","keyStoreIdentifier":{"provider":"kmip","uid":"1","kmipServerName":["localhost"],"kmipPort":{"$numberInt":"8081"},"keyWrapMethod":"encrypt"},"encryptedKey":{"$binary":{"base64":"WAAAAAVpdgAQAAAAAFSuxGWGC88Yi0dVi4QQTmYFa2V5ADAAAAAAuKH7TQXBSfyA/Oy9EGSnjkZ8+ooRlevbg6rfN34h/KNAN1fEo/eOb5tufQTItgcRAA==","subType":"0"}},"MAC":"JtRlaXS91rD79M4ZAQAAAAAAAAAAAAAA","auditRecordType":"header"}
```

Comment:
this is using KMIP Encrypt wrap method (see on this line "keyWrapMethod":"encrypt")